bfdd/bfd.c

```diff
@@ -317,8 +332,9 @@ void ptm_bfd_sess_up(struct bfd_session *bfd)
 	}
 }
 
-void ptm_bfd_sess_dn(struct bfd_session *bfd, uint8_t diag)
+void ptm_bfd_sess_dn(struct bfd_session *bfd, uint8_t diag, uint8_t peer_state)
```

Comment:
Please, don't change this function signature. Don't you already have that new information in `struct bfd_session`?